packages/app-cli/app/command-keymap.ts

```diff
@@ -0,0 +1,49 @@
+import BaseCommand from './base-command';
+import app from './app';
+import { _ } from '@joplin/lib/locale';
+const { cliUtils } = require('./cli-utils.js');
+
+// eslint-disable-next-line @typescript-eslint/ban-types -- no args expected
+type Args = {};
```

Comment:
```suggestion
interface Args { }
```

An empty `interface` might work better here: An empty interface shouldn't require `eslint-disable-next-line`.

Alternatively, it may also be possible to replace `public override async action(_args: Args) {` with `public override async action() {`.

Reply:
Ah perfect - thank you for the suggestion